account.go

```diff
@@ -658,12 +658,90 @@ func CreateAccountXLMTransactionWithMemo(from SeedStr, to AddressStr, amount str
 }
 
 // AccountMergeTransaction creates a signed transaction to merge the account `from` into `to`.
+// It will also create the `to` account if it does not already exist. If it does exist, and if
+// the `from` account has custom assets, this transaction will include payments of all of the
+// custom assets, and deletion of the `from` account's trustlines. AccountMergeTransaction will
+// error if the `from` account has a balance for an asset that the `to` account does not "trust."
```

Comment:
Since account merge is stellar terminology I think it'd be clearer to leave `AccountMergeTransaction` and add this functionality to either a new method or `RelocateTransaction`

Reply:
Relatedly, I don't think `AccountMergeTransaction` is used in client which is probably why it's missing timebounds.

Reply:
@mlsteele relocate transaction looked like it was strictly for relay payments, which is why i didn't really want to futz with it. and you are correct that it's not used in the client. that's coming soon. timebounds: tell me more about that.

Reply:
Ok I'm convinced.

For timebounds see `tb` [in client](https://github.com/keybase/client/blob/master/go/stellar/trustlines.go#L74) and `getTimeboundsForSending`.